restservice/src/main/java/org/itevents/dao/mybatis/sql_session_dao/UserMyBatisDao.java

```diff
@@ -42,7 +44,17 @@ public User getUserByName(String name) {
 
     @Override
     public void addUser(User user, String password) {
-        getSqlSession().insert("org.itevents.dao.mybatis.mapper.UserMapper.addUser", new UserPassword(user, password));
+        try {
+            getSqlSession().insert("org.itevents.dao.mybatis.mapper.UserMapper.addUser", new UserPassword(user, password));
+        } catch (Throwable e) {
+            Throwable t = e;
+            while (t.getCause() != null) {
+                t = t.getCause();
+                if (t instanceof SQLException) {
+                    throw new EntityAlreadyExistsDaoException(t.getMessage(), t);
+                }
+            }
+        }
```

Comment:
this is one of the antipatterns described in refactor task.
I will add TODO and refactor this within that task, ok?